src/stp/new_plays/BallPlacementThem.cpp

```diff
@@ -0,0 +1,55 @@
+//
+// Created by jessevw on 24.03.20.
+//
+
+#include "stp/invariants/game_states/BallPlacementThemGameStateInvariant.h"
+#include "stp/new_roles/BallPlacer.h"
+#include "stp/new_roles/BallAvoider.h"
+#include "stp/new_plays/BallPlacementThem.h"
+
+namespace rtt::ai::stp::play {
+
+    BallPlacementThem::BallPlacementThem() : Play() {
+        startPlayInvariants.clear();
+        startPlayInvariants.emplace_back(std::make_unique<invariant::BallPlacementThemGameStateInvariant>());
+
+        keepPlayInvariants.clear();
+        keepPlayInvariants.emplace_back(std::make_unique<invariant::BallPlacementThemGameStateInvariant>());
+
+        roles = std::array<std::unique_ptr<Role>, rtt::ai::Constants::ROBOT_COUNT()>{
+                std::make_unique<role::BallPlacer>(role::BallPlacer("ball_avoider_0")),    std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_1")),
+                std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_2")),    std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_3")),
+                std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_4")),    std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_5")),
+                std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_6")),    std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_7")),
+                std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_8")),    std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_9")),
+                std::make_unique<role::BallAvoider>(role::BallAvoider("ball_avoider_10"))};
+    }
```

Comment:
You probably also want to assign a keeper in this play

Reply:
yup made an issue for it